model/itemTab.cc

```diff
@@ -209,37 +209,43 @@ namespace minsky
                     def=definition(*v);                      
                     pango.setMarkup("9999");
                     if (rank==0)
-                      { 
+                      {   
                         varAttribVals.clear();
                         varAttribVals.push_back(v->name());
                         varAttribVals.push_back(def);                    
                         varAttribVals.push_back(v->init());
                         varAttribVals.push_back(it->tooltip);
                         varAttribVals.push_back(it->detailedText);
-                        varAttribVals.push_back(to_string(v->sliderStep));
-                        varAttribVals.push_back(to_string(v->sliderMin));
-                        varAttribVals.push_back(to_string(v->sliderMax));
-                        varAttribVals.push_back(to_string(v->value()));
+                        varAttribVals.push_back(str(v->sliderStep));
+                        varAttribVals.push_back(str(v->sliderMin));
+                        varAttribVals.push_back(str(v->sliderMax));
+                        varAttribVals.push_back(str(v->value()));
+                        
+                        size_t vACtr=0;                           
```

Comment:
What does vACtr stand for? Something counter?

Reply:
Yes, it is the variable attribute counter. Not a very good name I'm afraid. It could have just been Ctr, I guess.